### ISSUE TITLE
create winget package

### ISSUE THREAD

**Issue by christianparpart** [contour-terminal/contour#393]
Relevant URLs

* https://github.com/microsoft/winget-cli/
* https://github.com/microsoft/winget-create

this should make installing contour on windows as simple as doing `winget install contour`

**Comment by mTvare6** [comment]
What about a homebrew tap(or a bottle in homebrew core)?

**Comment by christianparpart** [comment]
> What about a homebrew tap(or a bottle in homebrew core)?

Good morning. I i would love too. Maybe we can get this asking with the 0.2.0 release fine, too. Thanks for the reminder. :)

**Comment by mTvare6** [comment]
Maybe add a tracker issue for all supported package managers?

**Comment by uspasojevic96** [comment]
Well the main talk about homebrew and osx is at https://github.com/contour-terminal/contour/issues/276

**Comment by WSLUser** [comment]
If you're going to do winget then I would advise creating a package for chocolately and scoop, which unlike winget, are fully functional package managers capable of updating a package. Winget will get there eventually and it's still good idea to get packaged for them, just keep in mind there are other well established package managers for Windows as well. As for precedence, Windows Terminal is also available for all 3 and they are all mentioned in their README for how to install.

**Comment by christianparpart** [comment]
> If you're going to do winget then I would advise creating a package for chocolately and scoop, which unlike winget, are fully functional package managers capable of updating a package

I'm not going to support **all** package managers on day one of 0.2.0 release (just like I didn't for 0.1.x releases). I.e. I want to start with winget (for sure) and brew (for sure, too). But I cannot do so *before having release 0.2.0*. That must be done first. I'm on it (despite some current family concerns that are keeping me busy day'n'night!).

WRT Chocolaty / scoop.  To be honest, I am not too sure, Chocolaty more likely, but I see more of a future in winget. I'm sure my opinions are never hard-coded into my nerve-system, so things may change, but still, one package manager at a time :)

**Comment by christianparpart** [comment]
Initial version for `winget` (for Contour 0.2.0 RC 1): https://github.com/microsoft/winget-pkgs/pull/24985

**Comment by WSLUser** [comment]
We can break the other 2 package managers in separate issues. Just wanted to raise awareness here. I wouldn't suggesting trying to support them all for a particular release, it's simply more about making Contour more available to users who might prefer a particular install method when you have the opportunity to do so. 

Also, personal life matters more! People can continue contributing, but everybody needs to ensure they're ok personally. Do you what you need to do and when you're ready, we'll all be waiting for you.

**Comment by halr9000** [comment]
> Initial version for `winget` (for Contour 0.2.0 RC 1): [microsoft/winget-pkgs#24985](https://github.com/microsoft/winget-pkgs/pull/24985)

Looks like this was gonna happen, but it was never merged, if I'm reading it right? Just dropping in to say there is def interest by others! I am chatting now w/friend about terminals, and I'm like "cool, winget installed that one, any others?", and when winget didn't have yours, my thought was that of all apps, a terminal does need to be in the official package manager!! 

TIA for your work, cool project for sure.

**Comment by christianparpart** [comment]
Hey @halr9000, many thanks for your interest in Contour!

As far as Windows packaging is concerned, we do provide support for .MSI installer from our Github's release page. Also, there is a Scoop package available.

I personally would really love having Contour also in winget, but I didn't have the time to persue the work back in time. If anybody else is having the time resource to try again, please feel free to help us :)

Mind, we're currently at version 0.3.12 (since quite some months already) and 0.4.0 is about to happen any time soon(ish).

Maybe I'll find the time to get into winget again, hopefully this year, but certainly not next days/weeks - I first have to deal with MacOS packaging and minor bug fixes. :)

**Comment by halr9000** [comment]
> Hey @halr9000, many thanks for your interest in Contour!
> 
> As far as Windows packaging is concerned, we do provide support for .MSI installer from our Github's release page. Also, there is a Scoop package available.

Cool, I'll try the scoop path. Oh btw actual install process is a bit murky when you start from the website.  To see what I mean, go to https://contour-terminal.org/install/#windows-10-or-newer, where it says:

> Please download Contour for Windows (the .msi file) from our release page, and double click on it to install.

But there's no hyperlinks here to the MSI or the release page. I see links in the other sections, so I guess an oversight, but it does feel like a dead-end. :)  Mentioning scoop in this section would also be helpful.

**Comment by christianparpart** [comment]
I've been adding a link to the release page (latest release) in the Windows section. Thanks for your feedback.

Nevertheless, I'd very much welcome a winget package at some point, eventually (no need to rush it, but when we release 0.4.0 soon**ish**, it would be super cool then.

**Comment by M-A-X-I-N** [comment]
Apologies for necroing, but I do believe this issue can be marked as completed. A package was submitted and merged into the community repository [microsoft/winget-pkgs: contour-terminal/Contour](https://github.com/microsoft/winget-pkgs/tree/master/manifests/c/contour-terminal/Contour) 2025-04-05. Based on that I'd say that the only thing remaining here would be to potentially add `winget` to the installation instructions?
